packages/use-intl/test/react/useTranslations.test.tsx

```diff
@@ -147,6 +147,22 @@ it('applies a time zone when using a built-in format', () => {
   expectFormatted('date', 'short', '5/8/23');
 });
 
+it('applies a time zone when using a date skeleton', () => {
+  const now = new Date('2024-01-01T00:00:00.000+0530');
+  renderMessage(`{now, date, ::yyyyMdHm}`, {now}, undefined, {
+    timeZone: 'Asia/Kolkata'
+  });
+  screen.getByText('1/1/2024, 00:00');
+});
```

Comment:
This even fixes a bug that currently affects Format.JS: https://github.com/formatjs/formatjs/issues/4279